<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSessionQWidget</class>
 <widget class="QWidget" name="NewSessionQWidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tasker - Session</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>200</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="goalQLabel">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Goal: Play guitar for 1h3</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QWidget" name="TaskContainerQWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="taskQLabel">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Task:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="taskLineEdit">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Write, Draw, Music, etc</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="startTimerQPushButton">
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="backQPushButton">
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Perferences</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="HooksQGroupBox">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Hooks</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QComboBox" name="audioDeviceQComboBox">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>74</y>
           <width>100</width>
           <height>25</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>200</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Default Mic</string>
          </property>
         </item>
        </widget>
        <widget class="QCheckBox" name="keyboardQCheckBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>38</y>
           <width>100</width>
           <height>16</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>Keyboard</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="mouseQCheckBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>59</y>
           <width>100</width>
           <height>16</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>Mouse</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="audioQCheckBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>100</width>
           <height>16</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>Audio</string>
         </property>
        </widget>
        <widget class="QComboBox" name="audioBackendQComboBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>74</y>
           <width>100</width>
           <height>25</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>200</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Default Mic</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
